symphony-bdk-core/src/main/java/com/symphony/bdk/core/service/user/UserService.java

```diff
@@ -0,0 +1,394 @@
+    } catch (ApiException apiException) {
+      throw new ApiRuntimeException(apiException);
+    }
+  }
+
+  /**
+   * Update delegates assigned to an user.
+   *
+   * @param uid User Id
+   * @param delegatedUserId Delegated user Id to be assigned
+   * @param actionEnum Action to be performed
+   */
+  public void updateDelegatesAssignedToUser(@NonNull Long uid, @NonNull Long delegatedUserId, @NonNull DelegateAction.ActionEnum actionEnum) {
+    try {
+      DelegateAction delegateAction = new DelegateAction().action(actionEnum).userId(delegatedUserId);
+      userApi.v1AdminUserUidDelegatesUpdatePost(authSession.getSessionToken(), uid, delegateAction);
+    } catch (ApiException apiException) {
+      throw new ApiRuntimeException(apiException);
+    }
+  }
+
+  /**
+   * Get feature entitlements of an user.
+   *
+   * @param uid User Id
+   * @return List of feature entitlements of the user.
+   */
+  public List<Feature> getFeatureEntitlementsOfUser(@NonNull Long uid) {
+    try {
+      return userApi.v1AdminUserUidFeaturesGet(authSession.getSessionToken(), uid);
+    } catch (ApiException apiException) {
+      throw new ApiRuntimeException(apiException);
+    }
+  }
+
+  /**
+   * Update feature entitlements of an user.
+   *
+   * @param uid User Id
+   * @param features List of feature entitlements to be updated
+   */
+  public void updateFeatureEntitlementsOfUser(@NonNull Long uid, @NonNull List<Feature> features) {
+    try {
+      userApi.v1AdminUserUidFeaturesUpdatePost(authSession.getSessionToken(), uid, features);
+    } catch (ApiException apiException) {
+      throw new ApiRuntimeException(apiException);
+    }
+  }
+
+  /**
+   * Get status of an user.
+   *
+   * @param uid User Id
+   * @return Status of the user.
+   */
+  public UserStatus getStatusOfUser(@NonNull Long uid) {
+    try {
+      return userApi.v1AdminUserUidStatusGet(authSession.getSessionToken(), uid);
+    } catch (ApiException apiException) {
+      throw new ApiRuntimeException(apiException);
+    }
+  }
+
+  /**
+   * Update the status of an user
+   *
+   * @param uid User Id
+   * @param status Status to be updated to the user
+   */
+  public void updateStatusOfUser(@NonNull Long uid, @NonNull UserStatus status) {
+    try {
+      userApi.v1AdminUserUidStatusUpdatePost(authSession.getSessionToken(), uid, status);
+    } catch (ApiException apiException) {
+      throw new ApiRuntimeException(apiException);
+    }
+  }
+
+  /**
+   * Get user by id
+   *
+   * @param uid User Id
+   * @param local If true then a local DB search will be performed and only local pod users will be
+   *             returned. If absent or false then a directory search will be performed and users
+   *             from other pods who are visible to the calling user will also be returned.
+   * @return User found by uid
+   */
+  public UserV2 getUserById(@NonNull Long uid, @Nullable Boolean local) {
```

Comment:
For each method that accepts `local` as param: let's provide a second one without `local` (default value will be `false` according to our doc )